pytest_django/plugin.py

```diff
@@ -608,12 +609,13 @@ def mailoutbox(
     _dj_autoclear_mailbox: None,
 ) -> list[django.core.mail.EmailMessage] | None:
     """A clean email outbox to which Django-generated emails are sent."""
-    if not django_settings_is_configured():
```

Comment:
This change LGTM. If someone explicitly asked for this fixture and Django is not configured, either it is not meant to run and skip is good, or there's some misconfiguration and then `None` is not really expected.